content/_code-samples/use-tickets/py/use-tickets-multisig.py

```diff
@@ -0,0 +1,133 @@
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Generate a wallet and request faucet
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+print("Setting up all the signers' accounts via the testnet faucet, this may take a while...")
+signer_1_wallet = generate_faucet_wallet(client=client)
+signer_2_wallet = generate_faucet_wallet(client=client)
+signer_3_wallet = generate_faucet_wallet(client=client)
+
+# Set the list of accounts that are able to authorize transactions on behalf of our Account via a multi-sig transaction
+signers = [
+        SignerEntry(account=signer_1_wallet.classic_address, signer_weight=1),
+        SignerEntry(account=signer_2_wallet.classic_address, signer_weight=1),
+        SignerEntry(account=signer_3_wallet.classic_address, signer_weight=1)
+]
+
+# Display all the signers' account address
+print("\nSigners:")
+signer_int = 1
+for signer in signers:
+    print(f"{signer_int}. {signer}")
+    signer_int += 1
+
+# A multi-sig transaction is achievable by trusting a list of keys to authorize transactions on behalf of the account
+# Construct a SignerList transaction
+tx_set_signer_list = SignerListSet(
+    account=myAddr,
+    signer_quorum=3,
+    signer_entries=signers
+)
+
+# We'll set the signer_quorum to 3, each key will represent 1 signer weight
+# if 3 of the signers sign a particular transaction, it's an authorized transaction on the XRPL
+
+# Sign transaction locally and submit
+tx_set_signer_list_signed = safe_sign_and_submit_transaction(transaction=tx_set_signer_list, wallet=test_wallet, client=client)
+
+# Construct a TicketCreate transaction, 3 tickets will be created
+tx_create_ticket = TicketCreate(
+    account=myAddr,
+    ticket_count=3
+)
+
+# Sign transaction locally and submit
+tx_create_ticket_signed = safe_sign_and_submit_transaction(transaction=tx_create_ticket, wallet=test_wallet, client=client)
+
+# Get a Ticket Sequence
+get_ticket_sequence = client.request(AccountObjects(
+    account=myAddr,
+    type=AccountObjectType.TICKET
+))
+
+# Since we created 3 Tickets previously, you're able to choose which Ticket you're going to use
+ticket1_sequence = get_ticket_sequence.result["account_objects"][0]["TicketSequence"]
+ticket2_sequence = get_ticket_sequence.result["account_objects"][1]["TicketSequence"]
+ticket3_sequence = get_ticket_sequence.result["account_objects"][2]["TicketSequence"]
+
+print(f"\nTickets issued:\n"
+      f"Ticket 1: {ticket1_sequence}\n"
+      f"Ticket 2: {ticket2_sequence}\n"
+      f"Ticket 3: {ticket3_sequence}")
+
+ticket_sequence = int(input("\nPick 1 ticket sequence to use for your next multi-sig transaction: "))
+
+# Construct AccountSet Transaction using a Ticket
+tx_1 = AccountSet(
+    account=myAddr,
+    fee="1000",
+    sequence=0,
+    last_ledger_sequence=None,
+    ticket_sequence=ticket_sequence
+)
+
+autofilled_account_set_tx = autofill(tx_1, client, len(signers))
+
+# Sign 'tx_1' using all the keys on signers[]
```

Comment:
I'd recommend highlighting that having us sign the transactions ourselves is odd for real-world multisign use cases. Ex.
```suggestion
# Sign 'tx_1' using all the keys on signers[]
# In a real app, you would share this transaction with key holders for them to sign
# but since we own these accounts for test purposes, we can sign directly here.
```